### DISCUSSION TITLE
How and from where to download the latest version of Edge file?

### DISCUSSION THREAD

**Discussion by fizsef** [AdguardTeam/AdGuardHome#5738] [Q&A]
Can you tell me where to download the latest version of the Edge binary file? From the website https://snapcraft.io/adguard-home I don't understand how to download a file like in the release AdGuardHome_linux_mipsle_softfloat.tar.gz

![1](https://user-images.githubusercontent.com/74646605/232237195-00f13fee-03e3-4562-8c46-3e2ef86eb0dc.jpg)

**Comment by ainar-g** [comment]
If you're using Snap, you should run the command provided in that window.

For binary installations, see the [_Platforms_ page on the Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki/Platforms#packaged-releases).
